### PR TITLE
Runner cache configuration 

### LINKED ISSUE #3: [Runner] Runner cache configuration points to the wrong address
Template uses `ServerAddress = "{{ template "minio.fullname" . }}:9000"` while the minio service template sets `{{ template "fullname" . }}-svc` as the service name, which makes the ServerAddress set in the runner config.toml point to an nonexisting host.

### PR DESCRIPTION
**Pull request by eduardocardoso**
- Fixed `ServerAddress` setting
- Added `Insecure` option to config.toml to allow using http connection instead of https.

Fixes #3

### CODE REVIEW

File: runner/templates/configmap.yaml [hunk 1]
```diff
@@ -64,7 +64,8 @@ data:
       {{ if .Values.cache.enabled -}}
       [runners.cache]
         Type = {{ .Values.cache.type | quote }}
-        ServerAddress = "{{ template "minio.fullname" . }}:9000"
+        ServerAddress = "{{ template "minio.fullname" . }}-svc:9000"
```
Comment (lwolf) [inline runner/templates/configmap.yaml]: This line will break cases when `.Values.cache.serverAddress` is defined in config. What you want, is to change only the generated service name inside the template.

https://github.com/lwolf/kube-charts/blob/master/runner/templates/_helpers.tpl#L51

**Review by lwolf — CHANGES_REQUESTED** [review]

File: runner/templates/configmap.yaml (hunk 1 above)
Comment (eduardocardoso) [reply runner/templates/configmap.yaml]: I'll move this `-svc` into the _helpers.tpl file then.
What do you think of including the port number in the minio.fullname template? This way the user can set the port inside `Values.cache.serverAddress`

Comment (lwolf) [reply runner/templates/configmap.yaml]: I totally agree with you.
I don't like the idea of having the port hardcoded. Moreover I  I'm thinking about completely removing `minio.fullname` template. I created a ticket for that #4.
After removing the template option port will be required part of the  `.Values.cache.serverAddress`.
But this requires a testing of the gitlab chart and it's much bigger than this pull request.

Comment (lwolf) [reply runner/templates/configmap.yaml]: If you willing to make a pull request for that ticket, it will be really great.

**Pull request merged**